invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/EItemsSearch/config.js

```diff
@@ -0,0 +1,34 @@
+import { invenioConfig } from '../../../common/config';
```

Comment:
Are there any entries here that can be moved to the general config? I am under the impression `resultsPerPageValues` appears a lot and its always the same.

Reply:
I agree but I don't know if we gonna need to have different config per page in the end.